dds/src/implementation/rtps/history_cache.rs

```diff
@@ -43,18 +43,17 @@ impl RtpsWriterCacheChange {
             _ => todo!(),
         };
 
-        DataSubmessageWrite {
-            endianness_flag: true,
-            inline_qos_flag: true,
+        DataSubmessageWrite::new(
+            true,
```

Comment:
This true and false are a little confusing (at least without IDE). Could they maybe get a variable name?

Reply:
Yes, they indeed should get a variable name. Alternatively one could invent extra types for each flag.